dashboard/src/shared/Kube.ts

```diff
@@ -136,12 +136,16 @@ export class Kube {
     verb: string,
     namespace: string,
   ) {
-    const { data } = await axiosWithAuth.post<{ allowed: boolean }>(url.backend.canI(cluster), {
-      group,
-      resource,
-      verb,
-      namespace,
-    });
-    return data.allowed;
+    try {
```

Comment:
Try-catch this call to prevent ugly errors. It's unlikely to happen, but since I'm using the `proxy` option in the package.json, I eventually ran into weird connectivity errors.